packages/riipen-ui/src/components/Editor.jsx

```diff
@@ -186,11 +198,12 @@ class Editor extends React.Component {
 
   getLinkedStyles = () => {
     const theme = this.context;
+    const { controlPosition } = this.props;
```

Comment:
Should updating this component to be a function component be added as a tech debt ticket?

Reply:
I believe it is a good idea, but we should have a general epic for updating each of the components to function components

Reply:
I will add it to the 'sprint retro' list